Keas.Mvc/ClientApp/components/Access/AccessTable.tsx

```diff
@@ -28,7 +28,7 @@ export default class AccessTable extends React.Component<IProps, {}> {
             Details
           </Button>
         ),
-        Header: 'Access',
+        Header: 'Details',
```

Comment:
I don't know that you should have a column heading for the Details column. Looks a little funny.
Previous look:
![image](https://user-images.githubusercontent.com/469010/88210812-9aaca880-cc09-11ea-816f-b57ec66c8c73.png)

New Look:
![image](https://user-images.githubusercontent.com/469010/88210868-af893c00-cc09-11ea-9099-a8a7632eee94.png)

Reply:
If the Header is empty then the application will break. Not sure how to do it without adding something there

Reply:
@wongband How about an space?
Header: ' ',

Reply:
Wait actually, I was being silly putting ' ' as the header will work. Should I do that for all detail columns?

Reply:
Yes, I think that looks better. Keep the Actions header though.

Reply:
Just updated all columns